test/spec/actions/join.js

```diff
@@ -424,6 +424,29 @@ describe('iD.actionJoin', function () {
         expect(graph.hasEntity('w-2')).to.be.undefined;
     });
 
+    it('prefers to keep the oldest way', function () {
```

Comment:
Hmm is it just me or I can't get this entire test suite (actions/join) to work?
I did a short debugging and it seems that `['restriction', 'connectivity'].forEach` section is creating a problem.
@k-yle  can you please confirm this for me?

Reply:
Yea, I took a quick look at this, if you add the following code block to `test/index.html`:

```js
window.onerror = function (message, source, line, col) {
  console.error(
    "\n\n\nFailed to execute test file.\n",
    message + "\n",
    "  at " + source + ":" + line + ":" + col
  );
};
```

Then this error is printed:
```
SyntaxError: Invalid character: '`'`
  at file:///.../iD/test/spec/actions/join.js:100:0
```

Replacing the template-strings with a normal strings fixes that error, but the test suite still doesn't run...

_(in any case, if you remove the dodgy tests, then the one I added in this PR passes)_